dd-java-agent/instrumentation/spring-boot/src/main/java/datadog/trace/instrumentation/springboot/SBEntrySpanOriginInstrumentation.java

```diff
@@ -0,0 +1,28 @@
+package datadog.trace.instrumentation.springboot;
+
+import static java.util.Arrays.asList;
+
+import com.google.auto.service.AutoService;
+import datadog.trace.agent.tooling.InstrumenterModule;
+import datadog.trace.instrumentation.span_origin.EntrySpanOriginInstrumentation;
+import java.util.HashSet;
+import java.util.Set;
+
+@AutoService(InstrumenterModule.class)
+public class SBEntrySpanOriginInstrumentation extends EntrySpanOriginInstrumentation {
+  private static final String WEB_BIND_ANNOTATION = "org.springframework.web.bind.annotation.";
+
+  public SBEntrySpanOriginInstrumentation() {
+    super("spring-boot-span-origin-instrumentation");
```

Comment:
we usually drop the `instrumentation` from the name
```suggestion
    super("spring-boot-span-origin");
```